File: discover_more_python/github_most_popular.py
import urllib2

request_headers = {
  'User-Agent': 'Holberton_School',
  'Authorization': 'token bc38982215eef8560352730fdfc6ee82bd22beee'
}

response = urllib2.urlopen('https://api.github.com/search/repositories?q=language:python&sort=stars&order=desc')
html = response.read()
print html

File: discover_ruby/github_most_popular.rb
#!/usr/bin/ruby

require 'httpclient'

extheaders = {
  'User-Agent' => 'Holberton_School',
  'Authorization' => 'token e107aed5a9acf8df3246ffc645de998ef2046d5a'
}

http = HTTPClient.new
response = http.get_content('https://api.github.com/search/repositories?q=language:ruby&sort=stars&order=desc')
puts response
